src/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -190,10 +194,8 @@ const CreateScenarioDialog = ({
                 maxWidth={'sm'}
                 fullWidth={true}
                 disableBackdropClick >
-                <DialogTitle id="form-dialog-title" className={classes.dialogContent} >
-                    <Typography variant='subtitle1'>
-                        {t(CREATE_SCENARIO_DIALOG_TITLE_LABEL_KEY, 'Create alternative scenario')}
-                    </Typography>
+                <DialogTitle id="form-dialog-title">
+                  {t(CREATE_SCENARIO_DIALOG_TITLE_LABEL_KEY)}
```

Comment:
Could you add a default label ? (this will be useful if translations are missing for some languages)